assembly/duration.ts

```diff
@@ -204,12 +211,15 @@ export class Duration {
 
   add<T = DurationLike>(durationToAdd: T, relativeTo: PlainDateTime | null = null): Duration {
     const duration = Duration.from(durationToAdd);
-    const largestUnit = larger(this.largestDurationUnit, duration.largestDurationUnit);
-  
+    const largestUnit = min<i32>(this.largestDurationUnit, duration.largestDurationUnit);
```

Comment:
Would `min<TimeComponent>` make it more or less clear?

Reply:
what do you mean? Just use `min(this.largestDurationUnit, duration.largestDurationUnit)`?

Reply:
That's even better